packages/server/src/utilities/index.ts

```diff
@@ -58,57 +58,32 @@ export function checkSlashesInUrl(url: string) {
 export async function updateEntityMetadata(
   type: string,
   entityId: string,
-  updateFn: any
+  updateFn: (metadata: Document) => Document
```

Comment:
This function is called by the test automation endpoint and I noticed this any and wanted to get rid of it. While here, I did some light refactoring to make these functions easier to follow.